spec/features/contextual_navigation_spec.rb

```diff
@@ -242,14 +247,23 @@ def and_i_visit_that_page
   end
 
   def and_i_visit_that_page_by_clicking_on_a_step_by_step_link
-    visit "/contextual-navigation/page-with-contextual-navigation?step-by-step-nav=8ad782bd-8603-40eb-97c0-434cb22047cd"
+    visit "/contextual-navigation/page-with-contextual-navigation?step-by-step-nav=8ad999bd-8603-40eb-97c0-999cb22047cd"
```

Comment:
Why did the content_id change?

Reply:
The content item is always the same when generating random step by steps. I realised the previous test that we were doing was giving false positives. To make sure that we 100% see the active step by step I've set a unique ID. ref https://github.com/alphagov/govuk_publishing_components/pull/900/files#diff-6a1129fe0126f5e5082740817b714c2bR140

Reply:
Could you add that to the commit note?

Reply:
Done 👍